projectile.el

```diff
@@ -1419,7 +1421,8 @@ Only text sent to standard output is taken into account."
       (with-temp-buffer
         (shell-command command t "*projectile-files-errors*")
         (let ((shell-output (buffer-substring (point-min) (point-max))))
-          (split-string (string-trim shell-output) "\0" t))))))
+          (mapcar (lambda (pat) (string-remove-prefix "./" pat))
```

Comment:
I'm afraid we can't do this, as it's going to kill performance for any larger project. It's very important to avoid mapping over all the files in Elisp when doing `alien` indexing (it's pretty slow).

Reply:
Ok, then I can change default find command to `find . -type f | cut -c3- | tr '\n' '\0'`. Would that be ok?

Tried it on large project there is minimal difference:

```
$  find . -type f | wc -l
184034
$ time find . -type f -print0 > /dev/null

real	0m2,112s
user	0m0,102s
sys	0m0,998s
$ time find . -type f | cut -c3- | tr '\n' '\0'> /dev/null

real	0m2,126s
user	0m0,201s
sys	0m0,989s
```

Reply:
That looks reasonable.

Reply:
Ok, updated PR